cfme/infrastructure/provider/__init__.py

```diff
@@ -165,9 +165,8 @@ def _form_mapping(self, create=None, **kwargs):
     @cached_property
     def vm_name(self):
         return version.pick({
-            version.LOWEST: "VMs",
-            '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            version.LOWEST: "VMs and Instances",
+            '5.7.1': "Virtual Machines"})
```

Comment:
This should be ```'5.7'``` not ```5.7.1``` else the auto will fail for 5.7.0, though it is unlikely we should ever use it against it, we shouldn't break backwards compatibility.